external/avro/src/main/scala/org/apache/spark/sql/avro/AvroOptions.scala

```diff
@@ -36,18 +36,14 @@ class AvroOptions(
   }
 
   /**
-   * Optional schema provided by an user in JSON format.
+   * Optional schema provided by an user in JSON format. This option can be set when
+   * 1. for reading Avro, the expected schema for deserialization is compatible with
```

Comment:
This wording sounds a bit unclear to me.

> When reading, the provided Avro schema should be compatible with the Avro schema from the actual data.
> When writing, the provided Avro schema should be compatible with the Avro schema converted from Spark schema.

Is it correct?

Reply:
Yes. I was trying to explain the actual usage.
The doc is revised in my latest commit.